docs/source/index.rst

```diff
@@ -8,7 +8,7 @@ Cloud Custodian Documentation
 
 Cloud Custodian is a tool that unifies the dozens of tools and scripts
```

Comment:
Tagging @kapilt for a quick look over the changes to this file since they are the top level index.  Looks good though.  Only thing I'm not sure about is if the TOC depth we dump out is now too long?